Makefile

```diff
@@ -20,9 +20,11 @@ VENDOR_MM_SERVER_DIR ?= vendor/github.com/mattermost/mattermost-server/v6
 ENTERPRISE_HASH ?= $(shell cat enterprise_hash)
 TESTFLAGS = -mod=vendor -timeout 30m -race -v
 
+# We specify version for the build; it is the latest semantic version of the tags
+DIST_VER=v6.3.7
```

Comment:
I think this will make us to have one more manual step. What if we make a `git tag` command and filter only the 6.3.X here